tests/by-util/test_du.rs

```diff
@@ -1295,3 +1295,27 @@ fn test_du_inodes_blocksize_ineffective() {
             );
     }
 }
+
+#[test]
+fn test_du_inodes_total_text() {
+    let ts = TestScenario::new(util_name!());
+    let at = &ts.fixtures;
+
+    at.mkdir("d");
+    at.mkdir("d/d");
+
+    let result = ts.ucmd().arg("--inodes").arg("-c").arg("d").succeeds();
+
+    let output = result.stdout_str();
+    let lines: Vec<&str> = output.lines().collect();
```

Comment:
The `output` var is not really necessary:
```suggestion
    let lines: Vec<&str> = result.stdout_str().lines().collect();
```